tests/benchdnn/doc/knobs_attr.md

```diff
@@ -87,6 +87,8 @@ default no scales are applied which corresponds to `common` policy and `1.f`
 scale value. Supported values are:
   - `common`         corresponds to `mask = 0` and means a whole tensor will be
                      multiplied by a single `SCALE` value.
+  - `host_scalar`    same as `common`, but is using host-side scalar memory
+                     object to store the scale value.
```

Comment:
I object it's "the same". Let's word that it denotes a single host scalar when executed for GPU engine kind\.

Reply:
`common` is described as "corresponds to `mask = 0` and means a whole tensor will be multiplied by a single `SCALE` value.", which is exactly the same for `host_scalar` policy, could you elaborate what is different? 

I could reword as a "for this policy the whole tensor will be multiplied by a single `SCALE` value that is stored on the host always", but I'm not sure it changes meaning in any sense, so just want to double check.

Reply:
> but I'm not sure it changes meaning in any sense

It does to me, let's stick to it.

Reply:
Updated based on offline discussion :)